README.md

```diff
@@ -7,13 +7,13 @@ well as allowing other plugins to make use of git.
 
 
 ## Installation ##
-This plugin now requires at least **Java 7** (the latest version supporting Java 6 was `0.8.5`)
+This plugin now requires at least **Java 8** (the latest version supporting Java 7 was `0.9.3`)
```

Comment:
Two things:
* I think it would be useful to preserve the information about the latest version supporting Java 6
* To avoid confusion I think we should rewrite this to avoid saying "now".

So something like:

> As of 0.10.0 (not released yet) this plugin requires at least **Java 8**.
> The latest version supporting Java 7 was `0.9.3`, while the latest version supporting Java 6 was `0.8.5`

Reply:
+1